packages/retail-ui-extensions/src/extension-api/scanner-api/scanner-api.ts

```diff
@@ -0,0 +1,24 @@
+import type {RemoteSubscribable} from '@remote-ui/async-subscription';
+
+/** The scanner source the POS device supports */
+export type SupportedSource = 'camera' | 'external' | 'embedded';
```

Comment:
Should the partners even know about the `embedded` source? Or should we just create a word that works for both `external` and `embedded`? 

If we do want to have three types, perhaps we should rename to embedded to `pos-go` for 3P because embedded sounds ambiguous. 

Curious to hear what everyone else thinks about this.

Reply:
Totally, now I understand the intent of the array!

Reply:
@heltisace Although external and embedded scanners use the same listener, the availability for each type of scanner needs to be distinct. POS Go is technically the only current device that supports embedded scanning however that could change in the future.